sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/DigitalTwinsRestClient.cs

```diff
@@ -181,7 +181,7 @@ internal Response<string> GetById(string id, CancellationToken cancellationToken
             }
         }
 
-        internal async Task<Response<string>> UpdateAsync(string id, string patchDocument, string ifMatch = null, CancellationToken cancellationToken = default)
+        internal async Task<Response> UpdateAsync(string id, string patchDocument, string ifMatch = null, CancellationToken cancellationToken = default)
```

Comment:
What's generated by default? Can we get rid of the customized code altogether?

Reply:
By default, the ```patchDocument``` argument to this method takes an object and tries to serialize that object. Our library deliberately customizes this layer to not serialize this particular argument and to just take the payload directly as a string. As such, we still can't remove this customization